packages/core-utils/src/__tests__/time.js

```diff
@@ -21,4 +26,13 @@ describe("time", () => {
       expect(getCurrentTime("America/New_York")).toMatchSnapshot();
     });
   });
+
+  describe("getFormattedTime", () => {
+    const withoutSeconds = formatDuration(9401);
+    const withSeconds = formatDurationWithSeconds(9401);
+
+    expect(withSeconds).not.toEqual(withoutSeconds);
+    expect(withSeconds).toMatchSnapshot();
+    expect(withoutSeconds).toMatchSnapshot();
```

Comment:
Every test case must be contained within an `it` function. See recommended changes as follows:

```suggestion
    const durationInSeconds = 9401;

    it(`should correctly format ${durationInSeconds} seconds as a duration with seconds`, () => {
      expect(formatDurationWithSeconds(durationInSeconds)).toMatchSnapshot();
    });

    it(`should correctly format ${durationInSeconds} seconds as a duration without seconds`, () => {
      expect(formatDuration(durationInSeconds)).toMatchSnapshot();
    });
```

Reply:
Thanks for the suggestion on this one! Fixed in 6211bd1